### ISSUE TITLE
Focus Medium: Projectile is impossible to complete.

### ISSUE THREAD

**Issue by Jorvalt** [thephoenixlodge/Heavens-of-Sorcery#115]
One of the requirements for unlocking this node in the Thaumcraft Auromancy tree is to be hit by a Ghast or Blaze fireball. Electroblob's Wizardry changes the projectile and the only way to fix this is to disable the projectile change in the config, which is not default here.
So good luck doing this on a server.

**Comment by ackeri** [comment]
Seconded that this is an issue, but it isn't actually impossible. Not sure what works but I unlocked this research on my game.

**Comment by Jorvalt** [comment]
> Seconded that this is an issue, but it isn't actually impossible. Not sure what works but I unlocked this research on my game.

It shouldn't be unless you changed the Electroblob's config. Afaik it replaces the entity itself so there's no possible way to get it even if you spawn it in.

**Comment by ewanm89** [comment]
Yeah, Electrobob's code looks is on the entity spawn event, if it sees it is the vanilla fireball it then cancels that entity, thereby killing it, and spawns it's own entity.

Thaumcraft is not fully integrated, we have plans for how to fix this in the works.

**Comment by merphle** [comment]
Try hitting yourself with a fire charge thrown by a vanilla dispenser? It's been a while since I last Thaumcrafted, but I think this works as an alternative to the Ghast and Blaze attacks. Only question is whether the fire charge event is also caught by Electrobob.

**Comment by Jorvalt** [comment]
> Try hitting yourself with a fire charge thrown by a vanilla dispenser? It's been a while since I last Thaumcrafted, but I think this works as an alternative to the Ghast and Blaze attacks. Only question is whether the fire charge event is also caught by Electrobob.

As ewanm said, Electroblob's kills the vanilla fireball entity and replaces it. The fire charge in a dispenser will spawn a vanilla fireball, therefore this still doesn't work.
